### PR TITLE
1024.feature: Step-by-Step operator guide

### PR DESCRIPTION
**Pull request by blaisep**
This introduces an Operator Guide
The Operator guide is a series of tutorials to show how to build a Tahoe deployment step by step, using the CLI and learning the purpose of each feature along the way.

### CODE REVIEW

**Comment by blaisep** [comment]
@meejah this is a big PR, but it's just Docs and the only changes to upstream are some new lines in the index.rst.

**Comment by blaisep** [comment]
You can see a staged HTML doc at:
file:///Users/bpabon/src/repos/tahoe/tahoe-lafs/docs/_build/html/examples/index.html

**Comment by meejah** [comment]
Needs a newsfragment too

File: docs/conf.py [hunk 1]
```diff
@@ -1,246 +1,26 @@
 # -*- coding: utf-8 -*-
-#
```
Comment (meejah) [inline docs/conf.py:2]: For this PR, can we leave the doc conf alone? (or move it to a new PR if changes are necessary?)
I don't _think_ this changes anything substantial, but ... (also I'd leave the comments in personally)

Comment (blaisep) [reply docs/conf.py:2]: @meejah yes, certainly, I will close this PR and prepare a new, content-only PR

**Comment by blaisep** [comment]
I'll close this PR to restructure into smaller, discrete changes.

**Pull request closed without merging**